src/dfmt/globmatch_editorconfig.d

```diff
@@ -2,8 +2,9 @@ module dfmt.globmatch_editorconfig;
 
 import std.path : CaseSensitive;
 import std.range : isForwardRange, ElementEncodingType;
-import std.string : isSomeChar, isSomeString, empty, save, front, popFront;
-import std.typecons : Unqual;
+import std.traits : isSomeChar, isSomeString;
+import std.range.primitives :  empty, save, front, popFront;
+import std.traits : Unqual;
```

Comment:
Do you need to `import std.traits` lines?

Reply:
No but I'd rather not change it since these changes should still keep their commit hashes.